<!DOCTYPE html>

<!-- ==============================
    Project:        Metronic "Acecv" Frontend Freebie - Responsive HTML Template Based On Twitter Bootstrap 3.3.4
    Version:        1.0
    Author:         KeenThemes
    Primary use:    Corporate, Business Themes.
    Email:          support@keenthemes.com
    Follow:         http://www.twitter.com/keenthemes
    Like:           http://www.facebook.com/keenthemes
    Website:        http://www.keenthemes.com
    Premium:        Premium Metronic Admin Theme: http://themeforest.net/item/metronic-responsive-admin-dashboard-template/4021469?ref=keenthemes
================================== -->
<html lang="en" class="no-js">
    <!-- BEGIN HEAD -->
    <head>

<style>
table, th, td {
  border: 1px solid black;
  text-align: center
  
}
</style>
	    
	<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-132371979-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());
  gtag('config', 'UA-132371979-1');
</script>

	<meta charset="utf-8"/>
	<script language="javascript">AC_FL_RunContent = 0;</script>
	<script src="AC_RunActiveContent.js" language="javascript"></script>
        <title>MBM Seminars - School of Mathematics, University of Leeds - 2019/2020</title>
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta content="width=device-width, initial-scale=1" name="viewport"/>
        <meta content="SMHD 2019" name="description"/>
        <meta content="Martin Lopez-Garcia" name="author"/>
	<meta name="keywords" content="Mathematical Biology and Medicine,Seminar,Applied Mathematics, Leeds, Health and Disease,Martin Lopez-Garcia,Martin,Lopez-Garcia,
				       University of Leeds,School of Mathematics,Leeds">

        <!-- GLOBAL MANDATORY STYLES -->
        <link href="http://fonts.googleapis.com/css?family=Hind:300,400,500,600,700" rel="stylesheet" type="text/css">
        <link href="vendor/simple-line-icons/css/simple-line-icons.css" rel="stylesheet" type="text/css"/>
        <link href="vendor/bootstrap/css/bootstrap.min.css" rel="stylesheet" type="text/css"/>

        <!-- PAGE LEVEL PLUGIN STYLES -->
        <link href="css/animate.css" rel="stylesheet">
        <link href="vendor/swiper/css/swiper.min.css" rel="stylesheet" type="text/css"/>

        <!-- THEME STYLES -->
        <link href="css/layout.min.css" rel="stylesheet" type="text/css"/>

        <!-- Favicon -->
	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
		    
        <link rel="shortcut icon" href="favicon.ico"/>
	<link rel="stylesheet" href="/assets/main.css">
        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
        <link rel="stylesheet" href="https://cdn.rawgit.com/jpswalsh/academicons/v1.8.5/css/academicons.min.css">
        <link rel="canonical" href="http://localhost:4000/">
    </head>
    <!-- END HEAD -->

    <!-- BODY -->
    <body id="body" data-spy="scroll" data-target=".header">

        <!--========== HEADER ==========-->
        <header class="header navbar-fixed-top">
            <!-- Navbar -->
            <nav class="navbar" role="navigation">
                <div class="container">
                    <!-- Brand and toggle get grouped for better mobile display -->
                    <div class="menu-container js_nav-item">
                        <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".nav-collapse">
                            <span class="sr-only">Toggle navigation</span>
                            <span class="toggle-icon"></span>
                        </button>
                    </div>

                    <!-- Collect the nav links, forms, and other content for toggling -->
                    <div class="collapse navbar-collapse nav-collapse">
                        <div class="menu-container">
                            <ul class="nav navbar-nav navbar-nav-right">
                                <li class="js_nav-item nav-item"><a class="nav-item-child nav-item-hover" href="#mbm">MBM Seminars</a></li>
                        				<li class="js_nav-item nav-item"><a class="nav-item-child nav-item-hover" href="#venue">Venue</a></li>
				                        <li class="js_nav-item nav-item"><a class="nav-item-child nav-item-hover" href="#contact">Contact</a></li>
                            </ul>
                        </div>
                    </div>
                    <!-- End Navbar Collapse -->
                </div>
            </nav>
            <!-- Navbar -->
        </header>
        <!--========== END HEADER ==========-->

        <!--========== SLIDER ==========-->
        <div class="promo-block">
            <div class="container">
                <div class="row">
                    <div class="col-sm-6 sm-margin-b-60">
                        <div class="margin-b-30">
				<br><br><br><br><br>
                            <h1 class="promo-block-title"><center>MBM Seminars 2019/2020</center></h1>
                            <p class="promo-block-text">Mathematical Biology and Medicine</p>
			    <p class="promo-block-text">School of Mathematics, University of Leeds</p>
			    
                        </div>
                    </div>
                    <div class="col-sm-6">
                        <div class="promo-block-img-wrap">
                            <img class="promo-block-img img-responsive" src="img/mockup/Leeds.jpg" align="Avatar">
                        </div>
                    </div>
                </div>
                <!--// end row -->
            </div>
        </div>
        <!--========== SLIDER ==========-->

        <!--========== PAGE LAYOUT ==========-->
        

        <!-- Keynote speakers-->
        <div id="mbm">
           <div class="container content-lg">
                <div class="row">
		    <div class="col-sm-12">
                                    <h2>MBM Seminars</h2>
<b>Wednesday 2nd October 2019, 12:00-13:00</b>
<br>
<p style="margin-bottom:0cm;"><i>Stochastic Spatial Predator-Prey Models</i></p> <a href="http://www1.phys.vt.edu/~tauber/">Prof. Uwe C Tauber</a> (Department of Physics, Virginia Tech, USA) 
<br>
Stochastic, spatially extended models for predator-prey interaction display striking spatio-temporal structures. These spreading activity 
fronts induce persistent correlations between predators and prey that can be studied through field-theoretic methods. Introducing local 
restrictions on the prey population induces predator extinction. The critical dynamics at this continuous absorbing state transition are 
governed by the scaling exponents of directed percolation. I will also address the influence of spatially varying reaction rates: 
Fluctuations in rare favorable regions cause a remarkable increase in both predator and prey populations. Intriguing novel features are 
found when variable interaction rates are affixed to individual particles rather than lattice sites. The ensuing stochastic dynamics 
combined with inheritance rules causes rapid time evolution for the rate distributions, with however overall neutral effect on stationary
population densites. I will finally discuss noise-induced spontaneous pattern formation in systems with three cyclically competing species
akin to spatial rock-paper-scissors games.
<br>
<br>
<br>
<b>Wednesday 9th October 2019, 12:00-13:00</b>
<br>
<p style="margin-bottom:0cm;"><i>Approximating persistence time for SIS infections in heterogeneous populations</i></p> <a href="https://www.hw.ac.uk/staff/uk/macs/damian-clancy.htm">Prof. Damian Clancy</a> (Heriot-Watt University, UK) 
<br>
For a susceptible-infectious-susceptible (SIS) infection model in a heterogeneous population, I will show how to derive 
simple and precise estimates of mean persistence time, from an endemic state to extinction of infection. Heterogeneity may 
be in either levels of infectiousness or of susceptibility, as well as in the infectious period distributions.
<br>
<br>
<br>
<b>Wednesday 16th October 2019, 12:00-13:00</b> <b><font color="red">This seminar has been CANCELLED</font></b>
<br>
<p style="margin-bottom:0cm;"><i>Prioritising cancer therapeutic targets via CRISPR-Cas9 screens and multi-omics data integration</i></p> <a href="https://www.sanger.ac.uk/people/directory/iorio-francesco">Dr. Francesco Iorio</a> (Wellcome Sanger Institute, UK) 
<br>
The molecular features of a patient’s tumour impact clinical responses and can be used to guide therapy, leading to more effective treatments and reduced toxicity. Most patients however do not benefit from such targeted therapies in part due to limited knowledge of candidate targets. Lack of efficacy is a leading cause of the 90% attrition rate in oncology drug development, and fewer molecular entities to new targets are being developed. Unbiased strategies that effectively identify and prioritise oncology therapeutic targets are needed to improve success rates in drug development and to accelerate the development of new therapies.
We performed genome-scale CRISPR-Cas9 dependency screens in 324 human cancer cell lines from 30 cancer types and developed a data-driven framework to prioritise cancer therapeutic candidates. Furthermore, we systematically identified genomic biomarkers of gene essentiality and integrated these with target tractability information, through a dedicated bioinformatic pipeline designed on purpose. This allowed to nominate and prioritise promising therapeutic targets at a genome-scale, and generated a catalog of ~600 promising hits for specific tissues and genotypes, ranked according to their predicted therapeutic potential, based on multiple evidences. We experimentally validated in vivo one of the most promising targets predicted by our approach - WRN (Werner syndrome ATP-dependent helicase) - as a selective and potent therapeutic target for cancers with microsatellite instability (MSI) from multiple tissues.
Our data and results are public available <a href="https://score.depmap.sanger.ac.uk/">here</a>.
<br>
<br>
<br>
<b>Wednesday 23rd October 2019, 12:00-13:00</b>
<br>
<p style="margin-bottom:0cm;"><i>Spatial and spatio-temporal models to understand ecological processes</i></p> <a href="https://www.kent.ac.uk/mathematics-statistics-actuarial-science/people/2236/rodriguez-de-rivera-ortega-oscar">Dr. Oscar Rodriguez de Rivera Ortega</a> (University of Kent, UK) 
<br>
Climatic change is expected to affect forest development in the short term, as well as the spatial distribution of species in the 
long term. The aim of this presentation is to study ecological processes from different perspectives and to analyse the different 
approaches to this problem. We apply spatio-temporal models based on Gaussian processes, which have been successfully applied in other 
disciplines. In order to fit the Bayesian model, instead of the common Markov Chain Monte Carlo (MCMC) method, we have used the novel 
integrated nested Laplace approximation (INLA), through the Stochastic Partial Differential Equation (SPDE) approach. This approach 
can be easily implemented, providing good results with less computation than MCMC. This model can be expanded to the spatio-temporal 
domain, including new parameters that reflect linear, nonlinear, autoregressive or more complex behaviours.
Bayesian approach shows more flexibility, and the inclusion of the spatial field or the latent spatio-temporal process allows 
including residuals as a proxy for unmeasured variables. Spatial and spatio-temporal Bayesian models are a really interesting approach 
for the understanding of environmental dynamics, not only because of the possibility to develop and solve more complex problems but 
also because of the transparent implementation.
<br>
<br>
<br>
<b>Wednesday 30th October 2019, 12:00-13:00</b>
<br>
<p style="margin-bottom:0cm;"><i>Using Gaussian processes to infer pseudotime and branching from single-cell data</i></p> <a href="https://www.research.manchester.ac.uk/portal/Magnus.Rattray.html">Prof. Magnus Rattray</a> (The University of Manchester, UK) 
<br>
Gaussian processes are a flexible class of model for statistical modelling and have been applied extensively to spatial and
temporal data. I will provide a brief introduction to Gaussian process inference and describe some applications of Gaussian 
process models to single-cell genomics data. We have developed a scalable implementation of the Gaussian process latent 
variable model (GPLVM) that can be used for pseudotime estimation when there is prior knowledge about pseudotime, e.g. from 
capture times available in single-cell time course data. Other dimensions of the GPLVM latent space can then be used to model 
additional sources of variation, e.g. from branching of cells into different lineages. We have also developed a branching 
Gaussian process to explicitly model such branching and identify which genes are associated with cellular branching and 
whether they branch late or early. To scale up inference in these applications we use sparse variational Bayesian inference 
algorithms to deal with large matrix inversions and intractable likelihood functions, implemented using the gpflow library. 
<br>
<br>
<br>
<b>Wednesday 6th November 2019, 12:00-13:00</b>
<br>
<p style="margin-bottom:0cm;"><i>Two species competition with a public good and randomly switching carrying capacity</i></p> <a href=" ">Mr. Robert West</a> (University of Leeds, UK) 
<br>
In this talk I will describe the effect of a randomly switching carrying capacity on a well-mixed two-species competition model. 
Here, demographic noise is coupled with environmental noise (via a switching carrying capacity that drives the total population 
size). We will discuss how the (fixation) probability that the slow species prevails depends on the environmental switching 
rate and how it can be peaked at a nontrivial switching rate when more time is spent in one state (asymmetric switching). We 
also compare our results with those obtained under periodic variations and show that, under fast random switching, the 
leading order of the fixation probability is the same but the first-order corrections differ. If time permits I will also 
present some work on three species models with a randomly switching reaction rate, where again environmental variability can 
lead to different fixation results.
<br>
<br>
<br>
<b>Wednesday 13th November 2019, <font color="red">13:00-14:00</font></b>
<br>
<p style="margin-bottom:0cm;"><i> Direct and indirect feedbacks in translation</i></p> <a href="https://www.abdn.ac.uk/ncs/profiles/m.romano/">Prof. Maria Carmen Romano</a> (University of Aberdeen, UK) 
<br>
Translational control of gene expression plays an important role in many situations, offering the advantage of a rapid cellular 
response, as opposed to transcriptional control, when more permanent cellular changes are required. We discuss different types 
of direct feedback mechanisms at the level of translation and how they can lead to a large repertoire of dynamical behaviour 
in protein synthesis, including oscillations, step-change and bistability. Moreover, we discuss indirect feedback mechanisms 
in translation that restore the balance between translational demands and available cellular resources upon cell perturbations.
<br>
<br>
<br>
<b>Wednesday 20th November 2019, 12:00-12:45</b>
<br>
<p style="margin-bottom:0cm;"><i>An automated spectral clustering: theory and applications</i></p> <a href="https://github.com/luisacutillo78/">Dr. Luisa Cutillo</a> (University of Leeds, UK) 
<br>
<b>Background:</b> Data clustering can be reformulated in terms of a graph partitioning problem where the given set of data is represented as a graph. In this context, eigenvectors of the graph Laplacian are often used to obtain a new geometric representation of the original data set, which generally enhances cluster properties and improves cluster detection.
<br>
<b>Methodology:</b> In this work, we apply a bootstrap Algebraic MultiGrid AMG method, which constructs a set of vectors associated with the graph Laplacian. These vectors, referred to as algebraically smooth ones, span a low dimensional Euclidean space, which we use to represent the data for cluster detection. We show how the computed smooth vectors employed in the construction of an AMG operator accurately approximate the space in the lower portion of the spectrum of a preconditioned version of the graph Laplacian.
<br>
<b>Application and conclusion:</b> We discuss the results of the application of our approach on simulated and real data, in particular we explore data from single-cell RNA sequencing (scRNA-seq). The results are very promising when dealing with networks with medium and high modularities. We show that our method can serve as valid alternative to modularity-based methods, when there is no “ground truth”. Indeed it exploits theory of AMG methods to decide how many smooth vectors have to be computed to well represent the data, while for the more standard spectral clustering it is not as clear how many eigenvectors to choose in advance in order to get satisfactory clustering results. We finally observe that our method has a linear computational complexity and can be very competitive for large-scale data sets. 
<br>
<br>
<br>
<center><b> <font color="red">Seminars on Wednesday 27th November and 4th December have been cancelled as a result of the strike action.</font></b></center>

<br>		    
<br>
<br>
<b>Wednesday 11th December 2019, 12:00-13:00</b>
<br>
<p style="margin-bottom:0cm;"><i>TBC</i></p> <a href="https://www.kennedy.ox.ac.uk/team/audrey-gerard">Dr. Audrey G&eacuterard</a> (University of Oxford, UK) 
<br>
TBC
<br>
<br>
<br>		    
</div>
</div>
</div>
</div>

                    
<!-- End MBM -->

        <!-- Venue -->
        <div id="venue">
           <div class="container content-lg">
                <div class="row">
		    <div class="col-sm-12">
<!--                        <div class="section-seperator margin-b-70">
                            <div class="margin-b-30">
                                <div class="margin-b-30">-->
                                    <h2>Venue</h2>
MBM Seminars take place at the MALLs, School of Mathematics Level 8, University of Leeds. 
Wednesdays 12-1pm. Coffee and biscuits are available from 11:30 at 
the School of Mathematics common room, everyone is welcome.

</div>
</div>
</div>
</div>
<!-- End Venue -->

<div id="contact">
            <div class="bg-color-sky-light">
                <div class="container content-lg">
                    <div class="row margin-b-40">
                        <div class="col-sm-9">
                           <h2>Contact</h2>
                            If you have any questions regarding the MBM seminars, please send an email to Mart&iacuten L&oacutepez-Garc&iacutea.
                        </div>
                    </div>
                    <!--// end row -->

                    <div class="row">
                        <div class="col-md-3 col-xs-6 md-margin-b-30">
                            <h4>Location</h4>
                            Dr Mart&iacuten L&oacutepez-Garc&iacutea,
				Department of Applied Mathematics,
				School of Mathematics,
				University of Leeds
                        </div>
                        <div class="col-md-3 col-xs-6 md-margin-b-30">
                            <h4>Phone</h4>
                            +44 (0)113-343-8951
                        </div>
                        <div class="col-md-3 col-xs-6">
                            <h4>Email</h4>
                            <a href="mailto:#">m.lopezgarcia@leeds.ac.uk</a>
                        </div>
                    </div>
                    <!--// end row -->
                </div>
            </div>
        </div>
<!-- End Contact -->

        <!--========== END PAGE LAYOUT ==========-->


        <!-- Back To Top -->
        <a href="javascript:void(0);" class="js-back-to-top back-to-top">Top</a>

        <!-- JAVASCRIPTS(Load javascripts at bottom, this will reduce page load time) -->
        <!-- CORE PLUGINS -->
        <script src="vendor/jquery.min.js" type="text/javascript"></script>
        <script src="vendor/jquery-migrate.min.js" type="text/javascript"></script>
        <script src="vendor/bootstrap/js/bootstrap.min.js" type="text/javascript"></script>

        <!-- PAGE LEVEL PLUGINS -->
        <script src="vendor/jquery.easing.js" type="text/javascript"></script>
        <script src="vendor/jquery.back-to-top.js" type="text/javascript"></script>
        <script src="vendor/jquery.smooth-scroll.js" type="text/javascript"></script>
        <script src="vendor/jquery.wow.min.js" type="text/javascript"></script>
        <script src="vendor/jquery.parallax.min.js" type="text/javascript"></script>
        <script src="vendor/jquery.appear.js" type="text/javascript"></script>
        <script src="vendor/swiper/js/swiper.jquery.min.js" type="text/javascript"></script>

        <!-- PAGE LEVEL SCRIPTS -->
        <script src="js/layout.min.js" type="text/javascript"></script>
        <script src="js/components/progress-bar.min.js" type="text/javascript"></script>
        <script src="js/components/swiper.min.js" type="text/javascript"></script>
        <script src="js/components/wow.min.js" type="text/javascript"></script>

    </body>
</html>
