<!DOCTYPE html>

<!-- ==============================
    Project:        Metronic "Acecv" Frontend Freebie - Responsive HTML Template Based On Twitter Bootstrap 3.3.4
    Version:        1.0
    Author:         KeenThemes
    Primary use:    Corporate, Business Themes.
    Email:          support@keenthemes.com
    Follow:         http://www.twitter.com/keenthemes
    Like:           http://www.facebook.com/keenthemes
    Website:        http://www.keenthemes.com
    Premium:        Premium Metronic Admin Theme: http://themeforest.net/item/metronic-responsive-admin-dashboard-template/4021469?ref=keenthemes
================================== -->
<html lang="en" class="no-js">
    <!-- BEGIN HEAD -->
    <head>

<style>
table, th, td {
  border: 1px solid black;
  text-align: center
  
}
</style>
	    
	<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-132371979-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());
  gtag('config', 'UA-132371979-1');
</script>

	<meta charset="utf-8"/>
	<script language="javascript">AC_FL_RunContent = 0;</script>
	<script src="AC_RunActiveContent.js" language="javascript"></script>
        <title>White Rose - Mathematical Biology Seminar Series - Universities of Leeds, Sheffield and York - 2020/2021</title>
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta content="width=device-width, initial-scale=1" name="viewport"/>
        <meta content="SMHD 2019" name="description"/>
        <meta content="Martin Lopez-Garcia" name="author"/>
	<meta name="keywords" content="Mathematical Biology and Medicine,Seminar,Applied Mathematics, Leeds, Health and Disease,Martin Lopez-Garcia,Martin,Lopez-Garcia,
				       University of Leeds,School of Mathematics,Leeds">

        <!-- GLOBAL MANDATORY STYLES -->
        <link href="http://fonts.googleapis.com/css?family=Hind:300,400,500,600,700" rel="stylesheet" type="text/css">
        <link href="vendor/simple-line-icons/css/simple-line-icons.css" rel="stylesheet" type="text/css"/>
        <link href="vendor/bootstrap/css/bootstrap.min.css" rel="stylesheet" type="text/css"/>

        <!-- PAGE LEVEL PLUGIN STYLES -->
        <link href="css/animate.css" rel="stylesheet">
        <link href="vendor/swiper/css/swiper.min.css" rel="stylesheet" type="text/css"/>

        <!-- THEME STYLES -->
        <link href="css/layout.min.css" rel="stylesheet" type="text/css"/>

        <!-- Favicon -->
	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
		    
        <link rel="shortcut icon" href="favicon.ico"/>
	<link rel="stylesheet" href="/assets/main.css">
        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
        <link rel="stylesheet" href="https://cdn.rawgit.com/jpswalsh/academicons/v1.8.5/css/academicons.min.css">
        <link rel="canonical" href="http://localhost:4000/">
    </head>
    <!-- END HEAD -->

    <!-- BODY -->
    <body id="body" data-spy="scroll" data-target=".header">

        <!--========== HEADER ==========-->
        <header class="header navbar-fixed-top">
            <!-- Navbar -->
            <nav class="navbar" role="navigation">
                <div class="container">
                    <!-- Brand and toggle get grouped for better mobile display -->
                    <div class="menu-container js_nav-item">
                        <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".nav-collapse">
                            <span class="sr-only">Toggle navigation</span>
                            <span class="toggle-icon"></span>
                        </button>
                    </div>

                    <!-- Collect the nav links, forms, and other content for toggling -->
                    <div class="collapse navbar-collapse nav-collapse">
                        <div class="menu-container">
                            <ul class="nav navbar-nav navbar-nav-right">
                                <li class="js_nav-item nav-item"><a class="nav-item-child nav-item-hover" href="#mbm">Seminar Series</a></li>
                        	<li class="js_nav-item nav-item"><a class="nav-item-child nav-item-hover" href="#venue">Venue</a></li>
				<li class="js_nav-item nav-item"><a class="nav-item-child nav-item-hover" href="#contact">Contact</a></li>
                            </ul>
                        </div>
                    </div>
                    <!-- End Navbar Collapse -->
                </div>
            </nav>
            <!-- Navbar -->
        </header>
        <!--========== END HEADER ==========-->

        <!--========== SLIDER ==========-->
        <div class="promo-block">
            <div class="container">
                <div class="row">
                    <div class="col-sm-6 sm-margin-b-40">
                        <div class="margin-b-30">
                            <p class="promo-block-text">White Rose Mathematical Biology Seminar Series</p>
			    <p class="promo-block-text">Universities of Leeds, Sheffield and York</p>
				<p class="promo-block-text">2021/2022</p>
                        </div>
                    </div>
                    <div class="col-sm-6">
                        <div class="margin-b-30">
				<br><br><br><br><br><br><br><!--<div class="promo-block-img-wrap">-->
                            <img class="promo-block-img img-responsive" src="img/mockup/WR-logo1.png" align="Avatar">
                        </div>
                    </div>
                </div>
                <!--// end row -->
            </div>
        </div>
        <!--========== SLIDER ==========-->

        <!--========== PAGE LAYOUT ==========-->
        

        <!-- Keynote speakers-->
        <div id="mbm">
           <div class="container content-lg">
                <div class="row">
		    <div class="col-sm-12">
                                    <h2>White Rose - Mathematical Biology Seminar Series</h2>
<br>
<br>
			    
			    <h2 style="color:red">Next seminar</h2>
<b>Wednesday 3rd November 2021, 13:00-14:00</b>
<br>
<p style="margin-bottom:0cm;"><i>Challenges and insights from modelling COVID-19 in a region where most cases are travel-related</i></p> <a href="https://ahurford.github.io/website/#dr.-amy-hurford"> Dr Amy Hurford (Memorial University of Newfoundland and Labrador, Canada)</a>
<br>
Many research groups have mechanistically modelled COVID-19 dynamics using variations on Susceptible-Infected-Recovered compartmental models. In the Canadian province of Newfoundland and Labrador, this approach is challenging to apply due to a high proportion of cases that are travel-related. In this talk, I will discuss our approach to modelling COVID-19 for Newfoundland and Labrador, and how some of the unique features of the province may have affected COVID-19 epidemiology and public health policy.
<br>
<br>
<br>
		    <h2 style="color:red">Forthcoming seminars</h2>
<b>Wednesday 10th November 2021, 13:00-14:00</b>
<br>
<p style="margin-bottom:0cm;"><i>A phylogenetic approach to unifying models in life and earth sciences</i></p> <a href="https://www.gzn.nat.fau.eu/2020/01/29/gastprofessorin-dr-rachel-warnock/"> Prof Rachel Warnock (Friedrich-Alexander-Universität Erlangen-Nürnberg, Germany)</a>
<br>
The fossilised birth-death (FBD) process provides a modelling framework that explicitly combines the diversification and fossil sampling processes. The model can be applied to infer dated trees, based on the analysis of phylogenetic (morphological or molecular) character data. It can also be used to infer key macroevolutionary parameters (origination, extinction and fossil recovery rates), even in the total absence of phylogenetic character data. Here, I will discuss my work into the application of the FBD model to recover metrics commonly used in quantitative paleobiology (origination, extinction, species richness) based on analyses of stratigraphic ranges or fossil occurrence data. The FBD model provides a mechanistic and flexible framework that has several key advantages compared to alternative methods, creating many opportunities for process-based inference in paleobiology.

<br>
<br>
<br>
<b>Wednesday 17th November 2021, 13:00-14:00</b>
<br>
<p style="margin-bottom:0cm;"><i>A data-driven perspective for the mathematical modeling of the COVID-19 pandemic</i></p> <a href="https://cosnet.bifi.es/people/yamir-moreno/"> Prof Yamir Moreno (Institute for Biocomputation and Physics of Complex Systems, University of Zaragoza, Spain)</a>
<br>
The Coronavirus disease 2019 (COVID-19) has forced an unprecedented response from health authorities worldwide and the World Health Organization. Despite the adoption of drastic measures, the pandemic is still ongoing worldwide, and recursive surges of infections are observed in many countries. Even with vaccination campaigns currently rolling out, specific pharmaceutical interventions need to be adopted to reduce the pressure on healthcare systems. Here we show results that correspond to different stages of the pandemic using data-driven modeling. Specifically, we present simulations using data-driven models tailored to mobility data from China, Spain, and the U.S. The models are used to estimate the effectiveness of customary public interventions on the spread of COVID-19 in these locations as well as to calculate herd immunity thresholds of realistic populations and vaccine coverage needed to protect them. Our main findings highlight that having a coordinated response system is key for the containment of the spread of COVID19 and its possible eradication at the lowest possible cost.
<br>
<br>
<br>
			    
<h1>Past seminars</h1>		    
			    
<b>Wednesday 21st October 2020, 13:00-14:00</b>
<br>
<p style="margin-bottom:0cm;"><i>Stochastic establishment of antibiotic resistance</i></p> <a href="https://www.ed.ac.uk/biology/groups/alexander">Dr. Helen Alexander</a> (University of Edinburgh, UK) 
<br>
The probability of establishment (i.e. non-extinction) of a lineage is a fundamental quantity in stochastic process models, and a classical question in population genetics. 
One practical application, where the possibility of stochastic extinction has been largely overlooked, is the emergence of antibiotic-resistant lineages in bacterial 
			    populations. Once prevalent, antibiotic resistance can be very difficult to eradicate and has become a major public health concern. But could 
			    we stop <i>de novo</i> resistance evolution at the source by promoting extinction of initially rare resistant mutants? In this project, we quantified the 
			    probability of establishment of resistant bacterial cells by fitting a simple stochastic model to experimental data. Our key finding was that 
			    antibiotic doses that are too low to clear an established resistant population may nonetheless prevent outgrowth from single cells with high 
			    probability. In this talk I will also explain some experimental design considerations when attempting to quantify stochastic processes, and 
			    briefly discuss plans for future work combining experiments and population dynamics models.
<br>
<br>
<br>
<b>Wednesday 28th October 2020, 13:00-14:00</b>
<br>
<p style="margin-bottom:0cm;"><i>The Architecture of Biofilms</i></p> <a href="https://www.dundee.ac.uk/people/fordyce-davidson">Prof. Fordyce Davidson</a> (University of Dundee, UK) 
<br>
Biofilms are social communities of microbial cells that underpin diverse processes including sewage bioremediation, plant growth promotion and plant protection, chronic 
infections and industrial biofouling. They are hallmarked by the production of an extracellular polymeric matrix. One of the phenotypic consequences of biofilm formation 
is that resident microbes are highly resistant to physical stresses and antimicrobial agents. Rapid advances in molecular and microscopy techniques are revealing a rich 
array of novel, complex behaviours associated with biofilm formation. For example new data of the authors reveals that the leading edge of a class of biofilms advances as 
a result of an extraordinary complex process that defies simple mechanical analogue. Behind this leading edge the biofilm matures differentially in response to environmental 
conditions. We propose that in order to keep pace with these rapid advances in experimental methods demands the application of new theoretical tools. We discuss some 
standard theoretical approaches used by the authors, highlighting their limitation by presenting images and movies that have changed our perception. We invite a discussion of 
novel perspectives on modelling biofilm formation that may yield a deeper and more useful understanding. As a starting point, we propose that one such approach may result 
from a holistic view that treats these complex cell-matrix composites as dynamically active materials.
<br>
<br>
<br>
<b>Wednesday 4th November 2020, 13:00-14:00</b>
<br>
<p style="margin-bottom:0cm;"><i>Adaptation and Self–Fertilisation</i></p> <a href="https://matthartfield.wordpress.com/">Dr. Matthew Hartfield</a> (Institute for Evolutionary Biology, The University of Edinburgh, UK)
<br>
Many organisms are hermaphrodites that are capable of self-fertilisation, where individuals produce both male and female sex cells that can fertilise one another. The degree of self-fertilisation affects the fixation of adaptive mutations; for example, selfers are more likely to fix recessive mutations than outcrossing organisms. Yet the effects of linked mutations on adaptive alleles, and the genetic footprint that beneficial mutations leave in genome sequence data obtained from self-fertilisation organisms, remain understudied topics. I will first discuss theoretical studies on how the spread of beneficial mutations affect the fixation of other linked selected alleles. Higher self-fertilisation rates increase the probability that linked deleterious alleles will fix alongside beneficial mutations, due to the resulting reduction in polymorphism and effective recombination. When considering a distribution of deleterious alleles, beneficial mutations generally need to be more recessive than predicted from single-locus results, for selfers to have higher overall fitness than outcrossers following a selective sweep. If recurrent adaptive mutation arises at a target site, then intermediate selfing rates maximize the fixation probability of linked recessive beneficial mutations. I will end by presenting results on the genetic footprint of adaptive mutations for different levels of dominance and self-fertilisation.
<br>
<br>
<br>
<b>Wednesday 11th November 2020, 13:00-14:00</b> <p style="color:red"> There is no seminar on this day. Next seminar on Wed 18th Nov.</p>
<br>
<br>
<b>Wednesday 18th November 2020, 13:00-14:00</b>
<br>
<p style="margin-bottom:0cm;"><i>CRISPR-cas9 screens and multi-omic data integration for identifying and new cancer therapeutic targets</i></p> <a href="https://www.humantechnopole.it/web/francesco.iorio">Dr. Francesco Iorio</a> (Human Technopole, Milan, Italy) 
<br>
Functional genomics approaches can overcome limitations -such as the lack of identification of robust targets and poor clinical efficacy- that hamper cancer drug development. 
We performed genome-scale CRISPR–Cas9 screens in 324 human cancer cell lines from 30 cancer types and developed a data-driven framework to prioritize candidates for cancer 
therapeutics. We integrated cell fitness effects with genomic biomarkers and target tractability for drug development to systematically prioritize new targets in defined 
tissues and genotypes. We verified one of our most promising dependencies, the Werner syndrome ATP-dependent helicase, as a synthetic lethal target in tumours from multiple 
cancer types with microsatellite instability. Our analysis provides a resource of cancer dependencies, generates a framework to prioritize cancer drug targets and suggests 
specific new targets. The principles described in this study can inform the initial stages of drug development by contributing to a new, diverse and more effective portfolio 
of cancer drug targets.
<br>
<br>
<br>
<b>Wednesday 25th November 2020, 13:00-14:00</b>
<br>
<p style="margin-bottom:0cm;"><i>Short-sighted viruses</i></p> <a href="https://www.bdi.ox.ac.uk/Team/katrina-lythgoe">Prof. Katrina Lythgoe</a> (University of Oxford, UK) 
<br>
With extremely short generation times and high mutability, many viruses can rapidly evolve and adapt to changing host environments. This ability allows viruses to evade 
host immune responses, evolve new behaviours, and exploit within-host ecological niches. However, natural selection typically generates adaptation in response to the 
immediate selection pressures that a virus experiences in its current host. Consequently, I will argue that some viruses, particularly those characterised by long durations 
of infection and ongoing replication, may be susceptible to short-sighted evolution, whereby a virus’ adaptation to its current host will be detrimental to its onward 
transmission within the host population. I will propose that viruses that are vulnerable to short-sighted evolution exhibit life history strategies that minimise its effects, 
and describe the various mechanisms by which this may be achieved. These concepts provide a new perspective on the way in which some viruses have been able to establish and 
maintain global pandemics.
<br>
<br>
<br>
<b>Wednesday 2nd December 2020, 13:00-14:00</b>
<br>
<p style="margin-bottom:0cm;"><i>Multiscale modelling of plants: Coupling between mechanics, chemistry and growth.</i></p> <a href="https://www.macs.hw.ac.uk/~mp91/">Dr. Mariya Ptashnyk</a> (Heriot-Watt University, UK) 
<br>
In multiscale modelling and analysis of the interplay between the mechanics, microscopic structure and the chemistry in plant tissues we shall assume that elastic properties of cell walls depend on the chemical processes, whereas chemical reactions depend on mechanical stresses within the cell walls. Numerical solutions for macroscopic models demonstrate heterogeneity in the cell wall displacement due to interactions between mechanical stresses, microstructure, and chemical processes. For plant growth we will consider macroscopic density-based models and microscopic description of growth processes on the plant cell level.
<br>
<br>
<br>
<b>Wednesday 9th December 2020, 13:00-14:00</b>
<br>
<p style="margin-bottom:0cm;"><i>Statistical Inference for epidemic models via likelihood approximation</i></p> <a href="https://www.nottingham.ac.uk/mathematics/people/philip.oneill">Prof. Philip O'Neill</a> (University of Nottingham, UK)
<br>
Individual-level stochastic models for infectious diseases invariably describe how the disease is transmitted from one individual to another. Conversely, in real life we rarely observe transmission, but instead we observe symptoms of disease. In many settings this complicates statistical inference because the likelihood of the observed data is intractable. Although methods such as data-augmented MCMC can deal with this, they can struggle in large-population settings. In this talk we describe a way to approximate the likelihood using interactions between pairs of individuals in the population.
<br>
<br>
<br>
			    <b>Wednesday 20th October 2021, 13:00-14:00</b>
<br>
<p style="margin-bottom:0cm;"><i>Understanding variation in malaria infection dynamics</i></p> <a href="https://eeb.utoronto.ca/profile/mideo-nicole/"> Dr Nicole Mideo (University of Toronto, Canada)</a>
<br>
Infection outcomes are highly variable: some individuals suffer severe illness while others seem relatively unharmed by the same infection. Underlying this variation 
			    are numerous sources of heterogeneity, including parasite genetics, host genetics, and infectious dose, among others. Yet mechanistic explanations 
			    of differential infection outcomes remain elusive. Focusing on data from experimental malaria infections in lab mice, my research has been 
			    developing and refining mathematical models to reveal those mechanistic explanations. In this talk, I will describe what we have learned about 
			    the parasite traits, host traits, and their interactions that give rise to the observed variation in malaria infection dynamics and outcomes 
			    for hosts. 
<br>
<br>
<br>
			    <b>Wednesday 27th October 2021, 13:00-14:00</b>
<br>
<p style="margin-bottom:0cm;"><i>Beyond the law of mass action in ecology: the effect of range-residency on encounter statistics</i></p> <a href="http://www.rmtzgarcia.com/pi---bio.html"> Dr Ricardo Martinez-Garcia (ICTP-South American Institute for Fundamental Research, Brazil)</a>
<br>
For over 100 years, mathematical models in population ecology have relied on very strong and unrealistic assumptions about the way individuals move and get to interact with each other and with the environment. Specifically, they assume that individuals behave like the molecules of an ideal gas: following completely random trajectories through the entire area occupied by the population and only interacting with each other when their trajectories intersect. Under these assumptions, the encounter rates follow the law of mass action, and individual encounter events are uniformly distributed within the population range.
<br>
However, mounting empirical evidence suggests that animals use space non-uniformly, occupy home ranges substantially smaller than the population range, and are often capable of nonlocal perception. I will discuss our recent efforts to develop a refined theory for ecological encounters grounded on empirically supported individual movement behavior [1, 2]. First, I will introduce the theoretical framework and derive novel analytical expressions for the encounter rates and the spatial distribution of encounters. Second, I will apply it to animal tracking data and discuss the ecological insights we obtain from such an analysis. I will conclude with a few remarks on future directions.
<br>
<br> 
References:
<br> 
			    [1] Martinez-Garcia, R., Fleming, C. H., Seppelt, R., Fagan, W. F., & Calabrese, J. M. (2020). How range residency and long-range perception change encounter rates. Journal of Theoretical Biology, 498, 110267. 
<br> 
			    [2] Noonan, M. J., Martinez-Garcia, R., Davis, G. H., Crofoot, M. C., Kays, R., Hirsch, B. T., ... & Calabrese, J. M. (2021). Estimating encounter location distributions from animal tracking data, Methods in Ecology and Evolution, 12 (7), 1158-1173.
<br>
<br>
<br>

</div>
</div>
</div>
</div>

                    
<!-- End MBM -->

        <!-- Venue -->
        <div id="venue">
           <div class="container content-lg">
                <div class="row">
		    <div class="col-sm-12">
<!--                        <div class="section-seperator margin-b-70">
                            <div class="margin-b-30">
                                <div class="margin-b-30">-->
                                    <h2>Venue</h2>
This seminar series takes place on-line via Zoom. If you would like to get the link to join, please send an email to the organizer, George Constable (george.constable@york.ac.uk).
</div>
</div>
</div>
</div>
<!-- End Venue -->

<div id="contact">
            <div class="bg-color-sky-light">
                <div class="container content-lg">
                    <div class="row margin-b-40">
                        <div class="col-sm-9">
                           <h2>Contact</h2>
                            If you have any questions regarding the seminar seminars, please send an email to George Constable (george.constable@york.ac.uk).
                        </div>
                    </div>
                    <!--// end row -->

<!--                    <div class="row">
                        <div class="col-md-3 col-xs-6 md-margin-b-30">
                            <h4>Location</h4>
                            Dr Mart&iacuten L&oacutepez-Garc&iacutea,
				Department of Applied Mathematics,
				School of Mathematics,
				University of Leeds
                        </div>
                        <div class="col-md-3 col-xs-6 md-margin-b-30">
                            <h4>Phone</h4>
                            +44 (0)113-343-8951
                        </div>
                        <div class="col-md-3 col-xs-6">
                            <h4>Email</h4>
                            <a href="mailto:#">m.lopezgarcia@leeds.ac.uk</a>
                        </div>
                    </div>
-->
                    <!--// end row -->
                </div>
            </div>
        </div>
<!-- End Contact -->

        <!--========== END PAGE LAYOUT ==========-->


        <!-- Back To Top -->
        <a href="javascript:void(0);" class="js-back-to-top back-to-top">Top</a>

        <!-- JAVASCRIPTS(Load javascripts at bottom, this will reduce page load time) -->
        <!-- CORE PLUGINS -->
        <script src="vendor/jquery.min.js" type="text/javascript"></script>
        <script src="vendor/jquery-migrate.min.js" type="text/javascript"></script>
        <script src="vendor/bootstrap/js/bootstrap.min.js" type="text/javascript"></script>

        <!-- PAGE LEVEL PLUGINS -->
        <script src="vendor/jquery.easing.js" type="text/javascript"></script>
        <script src="vendor/jquery.back-to-top.js" type="text/javascript"></script>
        <script src="vendor/jquery.smooth-scroll.js" type="text/javascript"></script>
        <script src="vendor/jquery.wow.min.js" type="text/javascript"></script>
        <script src="vendor/jquery.parallax.min.js" type="text/javascript"></script>
        <script src="vendor/jquery.appear.js" type="text/javascript"></script>
        <script src="vendor/swiper/js/swiper.jquery.min.js" type="text/javascript"></script>

        <!-- PAGE LEVEL SCRIPTS -->
        <script src="js/layout.min.js" type="text/javascript"></script>
        <script src="js/components/progress-bar.min.js" type="text/javascript"></script>
        <script src="js/components/swiper.min.js" type="text/javascript"></script>
        <script src="js/components/wow.min.js" type="text/javascript"></script>

    </body>
</html>
